packages/react/src/index.tsx

```diff
@@ -148,6 +148,8 @@ export class LoadUserError extends MicroappReactError {
   }
 }
 
+export * from './user-preferences/userPreferencesContext';
```

Comment:
➡️ Proposal | Always create `index.ts` file in folders; Also make sure that all folders and files are consistent with `dash-case`

```suggestion
export * from './user-preferences';
```